analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextBaseTest.cs

```diff
@@ -170,6 +170,92 @@ public void ProjectConfiguration_WhenInvalidXml_ThrowException()
            .WithMessage("File 'SonarProjectConfig.xml' has been added as an AdditionalFile but could not be read and parsed.");
     }
 
+    [TestMethod]
+    public void SonarLintFile_LoadsExpectedValues()
```

Comment:
Missing test case for Visual basic.

Reply:
You can make this test parametrized. More or less the same thing happens for C# and VB.NET. So, give it the file names and languages and the rest should be the same.